BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsEventParams.kt

```diff
@@ -0,0 +1,18 @@
+package com.braintreepayments.api
+
+public data class AnalyticsEventParams private constructor(
```

Comment:
```suggestion
data class AnalyticsEventParams private constructor(
```

The default modifier is public, so no need to specify `public` here.

Reply:
Thank you, addressed: [229c16b](https://github.com/braintree/braintree_android/pull/1031/commits/229c16bdaae38efcec18775a69b1c753c5e8715a)